From 17b7430b6b1b2e4b50369cb688fbfe3b1472aaef Mon Sep 17 00:00:00 2001
From: Zoltan HERPAI <wigyori@uid0.hu>
Date: Mon, 5 Jun 2023 17:57:15 +0200
Subject: [PATCH 4015/4018] sunxi: enable gmac on MYIR MYD-YT113X with the
 YT8531 PHY

The gmac is connected to a Motorcomm YT8531, for which the driver
has been picked from Starfive and ported over.

Support is not yet added in DTS, only for compile testing.

Signed-off-by: Zoltan HERPAI <wigyori@uid0.hu>
---
 configs/myir_myd_t113x_defconfig | 9 +++++++++
 1 file changed, 9 insertions(+)

--- a/configs/myir_myd_t113x_defconfig
+++ b/configs/myir_myd_t113x_defconfig
@@ -18,3 +18,12 @@ CONFIG_DRAM_SUNXI_TPR12=0x46
 CONFIG_DRAM_SUNXI_TPR13=0x34000100
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_OHCI_HCD=y
+CONFIG_PHY_MOTORCOMM=y
+CONFIG_SUN8I_EMAC=y
+CONFIG_RGMII=y
+CONFIG_RMII=y
+CONFIG_SUPPORT_EMMC_BOOT=y
+CONFIG_MMC_IO_VOLTAGE=y
+CONFIG_SPL_MMC_IO_VOLTAGE=y
+CONFIG_MMC_HS200_SUPPORT=y
+CONFIG_SPL_MMC_HS200_SUPPORT=y
